releasenotes/notes/build-images-from-ark-6eb0d5c8779bb970.yaml

```
@@ -0,0 +1,8 @@
+---
+features:
+  - |
+    Added support for building overcloud host images from Pulp package mirrors.
+    This feature can be toggled with the flag
+    ``stackhpc_build_overcloud_image_from_pulp_package_mirrors``. For the CI
+    Builder environment, this is enabled, so our StackHPC overcloud host images
+    will now be built with packages from Ark.
```

Minor rewording suggestions for clarity
```suggestion
    Added support for building overcloud host images from Pulp package mirrors.
    This feature can be toggled with the flag
    ``stackhpc_build_overcloud_image_from_pulp_package_mirrors``. By default,
    this feature is enabled in the ``ci-builder`` environment and disabled
    otherwise. StackHPC overcloud host images will now be built with
    packages from Ark.
```

---

i'd also change `will now be built with packages from Ark` to `are now built with packages from Ark` if you include new images in the PR

---

Done